src/Tizen.NUI.Components/Controls/TabView.cs

```diff
@@ -0,0 +1,144 @@
+    public class TabView : Control
+    {
+        private TabBar tabBar = null;
+
+        private TabContent content = null;
+
+        /// <summary>
+        /// Creates a new instance of TabView.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TabView()
+        {
+            Layout = new LinearLayout() { LinearOrientation = LinearLayout.Orientation.Vertical };
+            WidthSpecification = LayoutParamPolicies.MatchParent;
+            HeightSpecification = LayoutParamPolicies.MatchParent;
+
+            InitTabBar();
+            InitContent();
+        }
+
+        private void InitTabBar()
+        {
+            if (tabBar != null)
+            {
+                return;
+            }
+
+            tabBar = new TabBar();
+            tabBar.TabButtonSelected += tabButtonSelectedHandler;
+
+            Add(tabBar);
+        }
+
+        private void InitContent()
+        {
+            if (content != null)
+            {
+                return;
+            }
+
+            content = new TabContent();
+            content.WidthSpecification = LayoutParamPolicies.MatchParent;
+            content.HeightSpecification = LayoutParamPolicies.MatchParent;
+
+            Add(content);
+        }
+
+        private void tabButtonSelectedHandler(object sender, TabButtonSelectedEventArgs args)
+        {
+            if ((content != null) && (content.ViewCount > args.Index))
+            {
+                content.Select(args.Index);
+            }
+        }
+
+        /// <summary>
+        /// Gets TabBar of TabView.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TabBar TabBar
+        {
+            get
+            {
+                return tabBar;
+            }
+        }
+
+        /// <summary>
+        /// Gets TabContent of TabView.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TabContent Content
+        {
+            get
+            {
+                return content;
+            }
+        }
+
+        /// <summary>
+        /// Adds a tab with tab button and content view.
+        /// </summary>
+        /// <param name="tabButton">A tab button to be added.</param>
+        /// <param name="view">A content view to be added.</param>
+        public void AddTab(TabButton tabButton, View view)
+        {
+            if (TabBar != null)
+            {
+                TabBar.AddTabButton(tabButton);
+            }
+
+            if (Content != null)
+            {
+                Content.AddView(view);
+            }
+        }
+
+        /// <summary>
+        /// Removes a tab at the specified index of TabView.
+        /// </summary>
+        /// <param name="index">The index of TabView where a tab will be removed.</param>
+        /// <exception cref="ArgumentOutOfRangeException">Thrown when the index is less than 0, or greater than or equal to the number of tabs.</exception>
```

Comment:
could you check the Checker's warning?
=>
src/Tizen.NUI.Components/Controls/TabView.cs#L128
XML comment has cref attribute 'ArgumentOutOfRangeException' that could not be resolved

Reply:
I am sorry "using System;" was missing in TabView.cs for the warning.
I have added "using System;" in TabView.cs.